packages/compat/src/v1-app.ts

```diff
@@ -184,55 +183,35 @@ export default class V1App implements V1Package {
 
   @Memoize()
   babelConfig(): TransformOptions {
-    let plugins = get(this.app.options, 'babel.plugins') as any[];
-    if (!plugins) {
-      plugins = [];
-    } else {
-      plugins = plugins.filter(p => {
-        // even if the app was using @embroider/macros, we drop it from the config
-        // here in favor of our globally-configured one.
-        return (
-          !isEmbroiderMacrosPlugin(p) &&
-          // similarly, if the app was already using
-          // ember-cli-htmlbars-inline-precompile, we remove it here because we
-          // have our own always-installed version of that (v2 addons are
-          // allowed to assume it will be present in the final app build, the
-          // app doesn't get to turn that off or configure it.).
-          !TemplateCompiler.isInlinePrecompilePlugin(p)
```

Comment:
We can't just drop this filtering. It's incorrect for either `@embroider/macros` or `hbs` to run in stage1. 

If `@embroider/macros` run here it will do so with the wrong configuration.

If `hbs` runs here, we will lose visibility into component and template dependencies that come from within those `hbs` templates.

Reply:
Oops, removing this was actually by mistake. Good catch. I guess i screwed up a rebase or something.